src/main/kotlin/com/terraformation/backend/accelerator/api/ProjectModulesController.kt

```diff
@@ -21,20 +21,36 @@ import org.springframework.web.bind.annotation.RequestMapping
 import org.springframework.web.bind.annotation.RestController
 
 @AcceleratorEndpoint
-@RequestMapping("/api/v1/modules")
+@RequestMapping("/api/v1/projects/{projectId}/modules")
 @RestController
 class ProjectModulesController(
     private val moduleStore: ModuleStore,
 ) {
   @ApiResponse200
   @ApiResponse404
-  @GetMapping("/projects/{projectId}")
+  @GetMapping
   @Operation(summary = "Gets modules for a project.")
   fun listModules(
       @PathVariable("projectId") projectId: ProjectId,
   ): GetProjectModulesResponsePayload {
     val models = moduleStore.fetchModulesForProject(projectId)
-    return GetProjectModulesResponsePayload(models.map { ProjectModule(it) })
+    return GetProjectModulesResponsePayload(
+        models.mapIndexed { index, model -> ProjectModule(model, index) })
+  }
+
+  @ApiResponse200
+  @ApiResponse404
+  @GetMapping("/{moduleId}")
+  @Operation(summary = "Gets one module for a project.")
+  fun getModule(
+      @PathVariable("projectId") projectId: ProjectId,
+      @PathVariable moduleId: ModuleId,
+  ): GetProjectModuleResponsePayload {
+    val models = moduleStore.fetchModulesForProject(projectId)
```

Comment:
Cannot use fetch one here because we do need the list of modules for ordering. 

There is probably a SQL query that can do something like `join select row_num() from .... ` but it is pretty complex and will require changes to our data models.

Reply:
if the SQL method is preferred we can track in as a separate task.

Reply:
I guess you mean you need the index? What's that used for? Could the start date (which is unique and sortable) serve the same purpose?

Reply:
The start date is unique and sortable. I think this has something to do with how our client selector is set up. @nickgraz can you give some context?

Reply:
We need to know what the modules "title" is. For example "name: Kick-off, title: Module 1". When we load a module, we need to know what the title is, but the data isn't stored in a field it seems, it is derived from the position in the list instead.

Reply:
I'd kind of be inclined to say that the user-facing title of a module should be an explicit value we set in the spreadsheet, rather than something we calculate based on date ordering.

I could easily see us, say, wanting "Module 3A" and "Module 3B" at some point.

Reply:
Let me add the title in cohorts_modules table then.